src/Components/Rides/RiderSignup/RiderSignup.scss

```diff
@@ -11,4 +11,38 @@
 .apartment-radio-text-input {
   display: flex;
   flex-direction: row;
+}
+/* Pickup Location Image */
```

Comment:
nit: I don't think the comments are necessary here since they are just the css class.